application/F3DStarter.cxx

```diff
@@ -657,6 +657,8 @@ class F3DStarter::F3DInternals
       // clang-format off
       interactor.addBinding({ mod_t::NONE, "Left" }, "load_previous_file_group", "Others", std::bind(docString, "Load previous file group"));
       interactor.addBinding({ mod_t::NONE, "Right" }, "load_next_file_group", "Others", std::bind(docString, "Load next file group"));
+      interactor.addBinding({ mod_t::CTRL, "Left" }, "load_previous_file_group true", "Others", std::bind(docString, "Load previous file group while keeping the camera state"));
+      interactor.addBinding({ mod_t::CTRL, "Right" }, "load_next_file_group true", "Others", std::bind(docString, "Load next file group while keeping the camera state"));
```

Comment:
Can the doc be shorter ? This is increasing the width of the cheatsheet greatly.

Maybe:

`Load previous file group, keep camera`

Wdyt ?